cloudformation.yml

```diff
@@ -348,14 +360,15 @@ Resources:
     Properties:
       Handle:
         Ref: "WaitForInstanceWaitHandle"
-      Timeout: "900"
-      
+      Timeout: !Ref TimeoutCfStackCreation
+
    # We're allowing any ip address to access port 22, 14265, 14600 and 15600
    # SSH tunnel will be used to access iota-pm
-  InstanceSecurityGroup:
+  Ec2InstanceSecurityGroup:
     Type: "AWS::EC2::SecurityGroup"
     Properties:
-      GroupDescription: "Enable Access to our application via port 14265, 14600, 15600 and SSH access via port 22"
+      GroupName: "IOTA full node security group"
```

Comment:
The following resource(s) failed to create: [Ec2InstanceSecurityGroup]. . Rollback requested by user.
--
  | 22:11:07 UTC+0100 | CREATE_FAILED | AWS::EC2::SecurityGroup | Ec2InstanceSecurityGroup | IOTA full node security group already exists


Could you add AWS::StackName into the SG group name or any other variable ?